common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -193,6 +230,12 @@ def create_response_element(self, **kwargs):
             responseparam_element = etree.SubElement(response_element, 'responseparam')
             responseparam_element.set('type', 'tolerance')
             responseparam_element.set('default', str(tolerance))
+            if partial_range is not None and 'close' in credit_type:
+                responseparam_element.set('partial_range', str(partial_range))
+
+        if partial_answers is not None and 'list' in credit_type:
+            responseparam_element = etree.SubElement(response_element, 'responseparam')
```

Comment:
I should also mention...I admit I try to use it sparingly, but you can disable pylint violations inline. Very similar example in [this thread](https://groups.google.com/forum/#!topic/edx-code/qNk29Z5QnfA).